app/uk/gov/hmrc/economiccrimelevyenrolment/navigation/EclRegistrationDatePageNavigator.scala

```diff
@@ -60,7 +63,16 @@ class EclRegistrationDatePageNavigator @Inject() (
           case Some(_) =>
             allocateEnrolment(eclReferenceNumber, eclRegistrationDateString)
               .map(_ => routes.ConfirmationController.onPageLoad())
-          case _       => Future.successful(routes.NotableErrorController.detailsDoNotMatch())
+          case _       =>
+            auditConnector.sendExtendedEvent(
+              ClaimEnrolmentDetailsMismatchAuditEvent(
+                internalId,
+                ClaimEnrolmentDetailsMismatchReason.EclRegistrationDateMismatch,
+                eclReferenceNumber,
+                Some(eclRegistrationDateString)
```

Comment:
instead of using this string, pls use the original date with `toString` because BASIC_ISO_DATE is purely for matching purposes, the `toString` on the date will produce like `2007-12-03` which is what we need for the audit event

```suggestion
                Some(eclRegistrationDate.toString)
```

Reply:
Done :)